### DISCUSSION TITLE
How to serialize builtins.Index object?

### DISCUSSION THREAD

**Discussion by JRatschat** [quickwit-oss/tantivy-py#196] [Q&A]
Hi all,

I am currently developing a Dash app and I want to cache the builtins.Index object which is created with the tantivy.Index() command. In order to cache this object, I would need to serialize it with for example json or pickle, which does not seem to be possible. Has anyone of you experience with serializing the object?

Thanks!

**Comment by cjrh** [comment]
Related: https://github.com/PyO3/pyo3/issues/100
